### ISSUE TITLE
Error Tests

### ISSUE THREAD

**Issue by FintanH** [radicle-dev/radicle-surf#77]
[This](https://rust-lang.github.io/api-guidelines/documentation.html#examples-use--not-try-not-unwrap-c-question-mark) is soooo much better than using `expect` and `unwrap` in tests. Let's clean them up making our docs a lot better!

**Comment by sonro** [comment]
Hi there, I would like to help with this. I see you don't have a contributing guide or a licence, are there any speical requirements you have before I begin a pull request?

**Comment by FintanH** [comment]
Hey @sonro :wave: Really glad you'd like to help :)

Contribution guidelines are not up yet, but we have a ticket for it: https://github.com/radicle-dev/radicle-surf/issues/33. If you notice anything that's difficult to get set up or run it would be awesome to have it tracked in that issue! Sometimes it's hard to see the wood from the trees.

Regarding the license. We have marked it as GPL-3.0-only in the [Cargo.toml](https://github.com/radicle-dev/radicle-surf/blob/master/Cargo.toml#L10). There are a few more steps to take which I just added a ticket for here: https://github.com/radicle-dev/radicle-surf/issues/85.

Thanks for bringing these up, and let me know if I can do anything to help :v:
